.devcontainer/Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM mcr.microsoft.com/devcontainers/base:jammy
```

Comment:
Maybe this folder got generated automatically by mistake? I would like to avoid having a devcontainer config for the root of this project.

Reply:
I might have generated it myself so I could run the plugin in a devcontainer of its own

Reply:
Is there a best practice for this? I have a devcontainer config at the root of the plugins repo in order to generate an environment to run the `repo_manager` script. I also have a devcontainer config in the plugin directory to generate an environment specifically for plugin development.

Reply:
You only need python and poetry installed on your computer to run the script. I guess we could have a dev container for that but I'd rather not. The plugin environment is a lot more complicated and benefits more from the dev container env.

Reply:
I can remove that root config then